src/components/SelectInput/SelectInput.jsx

```diff
@@ -347,7 +348,7 @@ const SelectInput = ({
         isMulti={isMulti}
         isClearable={isClearable}
         aria-label={labelDefined}
-        aria-labelledby={labelledby}
+        aria-labelledby={!label ? labelledby : null}
```

Comment:
```suggestion
        aria-labelledby={!label ? labelledby : undefined}
```

Mettendo undefined l'attributo non viene proprio messo, che se capisco bene è la cosa giusta in questo caso